remoto/log.py

```diff
@@ -12,7 +12,9 @@ def reporting(conn, result, timeout=None):
         try:
             received = result.receive(timeout)
             level_received, message = list(received.items())[0]
-            log_map[level_received](message.strip('\n'))
+            message = message.splitlines()
```

Comment:
Actually, you can't remove `^M` like that, but you still need to strip it:

    message.strip('\n\r')

Also I wouldn't mind a test demonstrating the issue (and fix). Thanks!